### DISCUSSION TITLE
keep current column selected also selected in FX

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6576] [Ideas]
**Is your feature request related to a problem? Please describe**
when you select the column on xsheet, the element in the fx schematic does not show the white bars like it where not selected
just a yellow highlight

**Describe the solution you'd like**
select in xsheet and also in schematic
![67](https://user-images.githubusercontent.com/36406840/111883298-2c3b0880-8999-11eb-9f84-e13fb54ea6bd.png)

**Comment by DarrenTAnims** [comment]
Having just yellow text is a very small visual change. Often, you can't see which node is selected Perhaps if the background colour of the node changed or the selected node had a red outline, it would be more clear which node is selected.

**Comment by artisteacher** [comment]
The Xsheet and schematic are already synced, so a good deal of the functionality you want is already there. 

There's a white outline around the active node in the schematic that makes the selection very clear. However that outline only appears after clicking on an FX node and is only visible while the schematic itself is active. Paint with the brush, change rooms or etc. and the outline disappears.

**Comment by RodneyBaker** [comment]
A good idea.  Now to make it happen...
